src/ElmHtml/InternalTypes.elm

```diff
@@ -125,10 +139,61 @@ type HtmlContext msg
     = HtmlContext (List Tagger) (List Tagger -> EventHandler -> Json.Decode.Decoder msg)
 
 
+{-| Type for representing Elm's Attributes
+
+  - Attribute is an HTML attribute, like `Html.Attributes.title`
```

Comment:
It's a bit hard to tell from this explantation what attribute vs property is, but NBD